docs/appendices/release-notes/unreleased.rst

```diff
@@ -222,6 +222,9 @@ Administration and Operations
 - Added the ability to set a
   :ref:`storage_class <sql-create-repo-s3-storage_class>` for S3 repositories.
 
+- Updated to Admin UI version 1.24.4, updating to recent versions of engine.io,
```

Comment:
I think we shouldn't mention this as afaik this has no user-visible effect.

Reply:
+1

Reply:
I've removed the changelog item again. Thanks.